### PR TITLE
Feature/cbr 41/user profile model

### PR DESCRIPTION
**Pull request by PAKovalev**
Small, but important improvements related to user models

### CODE REVIEW

File: packages/app/src/stores/SessionStore/SessionStore.ts [hunk 1]
```diff
@@ -6,39 +6,34 @@ import {RequestModel, UserStatusEnum} from '@momentum-xyz/core';
...
 import {storage} from 'shared/services';
 import {api, FetchUserResponse} from 'api';
-import {UserProfileModel} from 'core/models';
+import {User} from 'core/models';
 import {LoginTypeEnum, StorageKeyEnum} from 'core/enums';
 import {guestProviderConfig, keycloakProviderConfig, web3ProviderConfig} from 'shared/auth';
 
 const SessionStore = types
   .model('SessionStore', {
+    userId: '',
```
Comment (Daboo1998) [inline packages/app/src/stores/SessionStore/SessionStore.ts:15]: Do we need userId? Maybe lets make it a view instead that returns `user.id`? Or remove it

**Review by Daboo1998 — APPROVED** [review]

File: packages/app/src/stores/SessionStore/SessionStore.ts (hunk 1 above)
Comment (PAKovalev) [reply packages/app/src/stores/SessionStore/SessionStore.ts:15]: Let's leave userId, it is non-null value.

Comment (Daboo1998) [reply packages/app/src/stores/SessionStore/SessionStore.ts:15]: But you asking response.id to it and also assign response to user, so why do we need this duplication?

Comment (PAKovalev) [reply packages/app/src/stores/SessionStore/SessionStore.ts:15]: ahh, yeah, you are right.

File: packages/app/src/core/models/User/User.ts [hunk 2]
```diff
@@ -2,57 +2,54 @@ import {types, Instance, flow} from 'mobx-state-tree';
...
       return self.inviteRequest.isDone;
     }),
+    // TODO: Move to another model
     setInvited(invited: boolean) {
       self.invited = invited;
     }
   }));
 
-export interface UserProfileModelInterface extends Instance<typeof UserProfileModel> {}
+export interface UserModelInterface extends Instance<typeof User> {}
```
Comment (aminbksi) [inline packages/app/src/core/models/User/User.ts:53]: do we need `Model` here? or `UserInterface` only?

Comment (PAKovalev) [reply packages/app/src/core/models/User/User.ts:53]: Model for all models so that to avoid misundestanding

**Review by aminbksi — APPROVED** [review]

**Review by dmitry-yudakov — APPROVED** [review]

**Pull request merged**